### PR TITLE
feat(component): add new  Progress Bar component

### LINKED ISSUE #129: Add Progress Bar component

### PR DESCRIPTION
**Pull request by luisbeqja**
fix #129

# What is it?

- [x] Feature / enhancement
- [ ] Bug
- [ ] Docs / tests

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

# Screenshots/Demo

immagine

# Checklist:

- [ ] My code follows the [developer guidelines of this project](https://github.com/qwikifiers/qwik-ui/blob/main/CONTRIBUTING.md)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: packages/headless/src/components/progress/progress.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { component$, QwikIntrinsicElements, Slot } from '@builder.io/qwik';
+
+export type ProgressProps = QwikIntrinsicElements['progress'];
+
+export const Progress = component$((props: ProgressProps) => {
+  return (
+    <progress {...props}>
+      <Slot />
```
Comment (gioboa) [inline packages/headless/src/components/progress/progress.tsx:8]: What is the purpose of this Slot?

**Comment by gioboa** [comment]
@all-contributors please add @luisbeqja for code

File: packages/headless/src/components/progress/progress.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { component$, QwikIntrinsicElements, Slot } from '@builder.io/qwik';
```
Comment (gioboa) [inline packages/headless/src/components/progress/progress.tsx:1]:
```suggestion
import { component$, QwikIntrinsicElements } from '@builder.io/qwik';
```

**Comment by luisbeqja** [comment]
I have read the CLA Document and I hereby sign the CLA

File: packages/headless/src/components/progress/progress.tsx (hunk 2 above)
Comment (gioboa) [inline packages/headless/src/components/progress/progress.tsx:1]:
```suggestion
import { component$, QwikIntrinsicElements } from '@builder.io/qwik';
```

**Pull request merged**